src/WPF/ArcGISRuntime.WPF.Samples/ArcGISRuntime.WPF.Samples.VBNet.vbproj

```diff
@@ -191,6 +193,8 @@
     <None Include="Samples\Layers\FeatureLayerUrl\metadata.json" />
     <None Include="Samples\Layers\RasterLayerImageServiceRaster\metadata.json" />
     <None Include="Samples\Layers\RasterLayerImageServiceRaster\README.md" />
+    <None Include="Samples\Layers\RasterLayerRasterFunction\metadata.json" />
+    <None Include="Samples\Layers\RasterLayerRasterFunction\README.md" />
```

Comment:
More readmes included; I suspect there are readmes in other samples as well. Maybe there should be an issue to remove them?